bin/e2e/delete-profile-user.ts

```diff
@@ -37,7 +37,7 @@ async function cleaningProfileStaging(): Promise<void> {
       process.env.CONNECT_TEST_ACCOUNT_SUB,
     );
     console.log("✅ Test user profile deleted");
-  } catch (error) {
+  } catch (error: any) {
```

Comment:
I guess `Error` does not work?

Reply:
With ts `4.2`, you can only type this as `any` or `unknown` :/.

Reply:
Also, in this case, it would not work, as there is no `response` key in `error` of type `Error`, and I don't have access to the error type from the OpenAPI client.